samples/create-symbol-styles-from-web-styles/src/main/java/com/esri/arcgismaps/sample/createsymbolstylesfromwebstyles/screens/CreateSymbolStylesFromWebStylesScreen.kt

```diff
@@ -0,0 +1,123 @@
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.toolkit.geoviewcompose.MapView
+import com.esri.arcgismaps.sample.createsymbolstylesfromwebstyles.components.CreateSymbolStylesFromWebStylesViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialog
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+
+/**
+ * Main screen layout for the sample app
+ */
+@Composable
+fun CreateSymbolStylesFromWebStylesScreen(sampleName: String) {
+
+    // Get a reference to the view model
+    val mapViewModel: CreateSymbolStylesFromWebStylesViewModel = viewModel()
+
+    // Keep track of legend visibility state
+    var showLegend by remember { mutableStateOf(false) }
+
+    Scaffold(topBar = { SampleTopAppBar(title = sampleName) }, content = {
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(it),
+        ) {
+            MapView(
+                modifier = Modifier.fillMaxSize(), arcGISMap = mapViewModel.arcGISMap,
+                // Hide the legend on any tap of the map view
+                onDown = { showLegend = false },
+                // Update the map scale in the view model
+                onMapScaleChanged = mapViewModel::onMapScaleChanged
+            )
+            // Show the FAB
+            if (!showLegend) {
+                FloatingActionButton(modifier = Modifier
+                    .align(Alignment.BottomEnd)
+                    .padding(bottom = 36.dp, end = 24.dp),
+                    // On click, show the legend
+                    onClick = { showLegend = true }) {
+                    Icon(Icons.Filled.Info, contentDescription = "Show legend button")
+                    Spacer(Modifier.padding(8.dp))
+                }
+                // Show the legend
+            } else {
+                Column(
+                    modifier = Modifier
+                        .wrapContentSize()
+                        .align(Alignment.BottomEnd)
+                        .padding(bottom = 36.dp, end = 12.dp)
+                        .background(Color.White)
+                ) {
+                    // For each pair of symbol name and icon
+                    for ((symbolName, symbolIcon) in mapViewModel.symbolNamesAndIconsFlow.collectAsState().value) {
+                        Row(
+                            modifier = Modifier
+                                .wrapContentSize()
+                                .padding(top = 4.dp, bottom = 4.dp, start = 8.dp, end = 8.dp),
+                        ) {
```

Comment:
By default, a modifier will `wrapContentSize` and the padding could be a bit simplified: 
```suggestion
                        Row(Modifier.padding(8.dp)) {
```